snapshot/bazel/constants.go

```diff
@@ -30,4 +30,7 @@ const (
 
 	// connection-limit fs_util flag setting
 	connLimit = 3
+
+	// max time execd fs_util commands can run before timing them out
+	fsUtilTimeoutSec = 1800
```

Comment:
this seems a little high. could we go down to ~10 min?

Reply:
I originally had this as 10 and then upped it - will take this as a sign that that was a better value